### ISSUE TITLE
Running Accessibility Insights for Windows against a Chromium based browser results in the browser frame elements being flagged with "Chromium components should be scanned with a  web-based scanner"

### ISSUE THREAD

**Issue by amandabaker** [microsoft/accessibility-insights-windows#1625]
Please check whether the bug has [already been filed](https://github.com/Microsoft/accessibility-insights-windows/issues).

This repo contains code for Accessibility Insights for Windows. We maintain code for its automated checks rule engine at [axe-windows](https://github.com/microsoft/axe-windows/issues/). Most false positive questions or rule clarifications belong in [axe-windows](https://github.com/microsoft/axe-windows/issues/new/choose), whereas questions about the Accessibility Insights tool UI should go here. If you aren't sure, it's definitely OK to use this repo. Thanks for contributing!

**Describe the bug**
When running Accessibility Insights for Windows against a Chromium based browser, all elements within the window are flagged with the error: "Chromium components should be scanned with a web-based scanner". This includes elements of the browser frame which are not built on web tech and cannot be tested with the web version of Accessibility Insights. 

The result of this bug is that it is very hard to navigate the accessibility tree to find errors since every element has an error and is marked with the red exclamation mark.

Instead, only the elements within web content should have this error, or there should be a way to disable this error so that the developers working on these browsers can easily pinpoint true errors when fixing accessibility bugs in the browser.

**To Reproduce**
1. Launch a Chromium-based browser (e.g. Edge)
2. Run a fast pass in Accessibility Insights for Windows against the browser window
3. Check errors

**Expected behavior**
Only web components are flagged with the error "Chromium components should be scanned with a web-based scanner"

**Actual behavior**
All elements within the browser window are marked with the error, including the browser frame itself. Note that the browser frame cannot and should not be tested with the web-based scanner.

**Desktop (please complete the following information):**
 - OS: Windows 11 22H2 
 - Accessibility Insights for Windows Version: 1.1.2334.1
 - Target Application: Edge
 - Target Application Version: 115.0.1880.0

**Screenshots:**
<img width="211" alt="image" src="https://github.com/microsoft/accessibility-insights-windows/assets/8602218/f74f5072-93d7-49ac-98e0-79c967b5b2a6">

**Additional context**
Priority requested -

**Comment by DaveTryon** [comment]
@amandabaker, we made this change because Accessibility Insights for Windows can basically say "Something is wrong with this HTML app", but it can't tell you what to do about it. Accessibility Insights for Web understands HTML and can point out specific options to address any problems that it finds. If you can render the HTML in a standalone browser where Accessibility Insights for Web is installed, you'll end up with far more actionable results.

Your best path forward is to isolate your HTML and render it in a browser where it can be inspected with Accessibility Insights for Web. Inside of Accessibility Insights for Windows, ignore _all_ results under the `BrowserRootView` window.

If you are unable to render the HTML separately, your second best option is to review the automated checks results in the Test view, then filter out the error about Chromium. Issues will still be reported against items inside the WebView control, but it will be _much_ harder to try to figure out what to change to address the reported issues--some of them may even be handled completely by the browser and therefore outside of your ability to directly influence.

Thanks for using Accessibility Insights!

**Comment by DaveTryon** [comment]
@amandabaker, a quick update: In Edge, the entire application (including tabs, menus, and even the system menu are rendered by Chromium. The UIA tree includes exactly 3 windows that are not rendered by Chromium. They are the outer frame (class name `Chrome_WidgetWin_1`), then 2 sibling panes with class name `Intermediate D3D Window`, and a third sibling with the class name of `BrowserRootView`. This last window draws the entire user experience, and it's all rendered in Chromium.

**Comment by amandabaker** [comment]
@DaveTryon The problem is that non-HTML components of the browser frame (e.g. "Min"/"Max"/"Close" buttons) also include this error:
<img width="98" alt="image" src="https://github.com/microsoft/accessibility-insights-windows/assets/8602218/0a1e7642-3034-4d32-aae0-cc0055bbd669">

Since these aren't written using HTML, we can't isolate the elements to test them using Accessibility Insights for Web. So while trying to fix accessibility bugs in the browser frame (I work on Edge), it's hard to isolate true errors from this noise since all the elements/views in the accessibility tree are flagged with errors. There are other ways to investigate the bug, but for those of us working on accessibility bugs in the browser (not in web content), this noise is a huge pain.

**Comment by DaveTryon** [comment]
We need to release Axe.Windows before this can be picked up in AIWin. We're hoping to merge https://github.com/microsoft/axe-windows/pull/922 before doing that, so we can fix both issues in a single release.

**Comment by DaveTryon** [comment]
@amandabaker, as of release https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.2360.01, we now limit the `ChromiumComponentShouldUseWebScanner` rule to elements within a Chrome Document (i.e., the HTML components)
